documentation/content/en/books/handbook/x11/_index.adoc

```diff
@@ -1092,84 +1089,75 @@ In this case, create an executable [.filename]#~/.xsession#:
 [[x11-wm-kde]]
 === KDE
 
-KDE is another easy-to-use desktop environment.
-This desktop provides a suite of applications with a consistent look and feel, a standardized menu and toolbars, keybindings, color-schemes, internationalization, and a centralized, dialog-driven desktop configuration.
-More information on KDE can be found at http://www.kde.org/[http://www.kde.org/].
-For FreeBSD-specific information, consult http://freebsd.kde.org/[http://freebsd.kde.org].
+KDE Plasma[https://kde.org/plasma-desktop/] is another easy-to-use desktop environment.
+It provides a suite of applications with a consistent look and feel, a standardized menu and toolbars, keybindings, color schemes, internationalization, and a centralized, dialog-driven desktop configuration.
+For more information on KDE, see https://www.kde.org/[https://www.kde.org/] and FreeBSD-specific https://freebsd.kde.org/[https://freebsd.kde.org].
 
-To install the KDE package, type:
+To install the KDE package:
 
 [source,shell]
 ....
 # pkg install x11/kde5
 ....
 
-To instead build the KDE port, use the following command.
-Installing the port will provide a menu for selecting which components to install.
-KDE is a large application and will take some time to compile, even on a fast computer.
+To instead build KDE, use the following command.
+You will be prompted to choose components.
+Building the required collection of ports will be time-consuming. 
 
 [source,shell]
 ....
 # cd /usr/ports/x11/kde5
 # make install clean
 ....
 
-KDE requires [.filename]#/proc# to be mounted.
-Add this line to [.filename]#/etc/fstab# to mount this file system automatically during system startup:
+KDE uses D-Bus. 
+D-Bus is automatically installed as a dependency.
+Edit [.filename]#/etc/rc.conf# for D-Bus to start when the system boots:
 
-[.programlisting]
-....
-proc           /proc       procfs  rw  0   0
-....
-
-KDE uses D-Bus for a message bus and hardware abstraction.
-These applications are automatically installed as dependencies of KDE.
-Enable them in [.filename]#/etc/rc.conf# so they will be started when the system boots:
-
-[.programlisting]
+[source,shell]
 ....
-dbus_enable="YES"
+# sysrc dbus_enable="YES"
 ....
 
-Since KDE Plasma 5, the KDE Display Manager, KDM is no longer developed.
-A possible replacement is SDDM.
-To install it, type:
+To install SDDM, the display manager that is most commonly used to start Plasma:
 
 [source,shell]
 ....
 # pkg install x11/sddm
 ....
 
-Add this line to [.filename]#/etc/rc.conf#:
+– and edit [.filename]#/etc/rc.conf#:
 
-[.programlisting]
+[source,shell]
 ....
-sddm_enable="YES"
+# sysrc sddm_enable="YES"
 ....
 
-A second method for launching KDE Plasma is to type `startx` from the command line.
-For this to work, the following line is needed in [.filename]#~/.xinitrc#:
+When SDDM appears: before you login, to avoid possible problems with Wayland, use the session menu to switch from _Plasma&#160(Wayland)_ to _Plasma&#160(X11)_. 
```

Comment:
Will this render properly? 

<https://asciidoc-py.github.io/faq.html#_how_can_i_include_non_breaking_space_characters> suggests `&#160` for non-breaking space.

Reply:
IIRC, & entities need a final semicolon.

Reply:
Ah, a copy-and-paste error. Thank you. (I learnt `&nbsp;` many years ago.)